src/tasks/export-data/export-data-v2/task.json

```diff
@@ -0,0 +1,95 @@
+{
+  "id": "3137A4C5-0E44-4D89-AC0C-BEEDEC036428",
+  "name": "PowerPlatformExportData",
+  "friendlyName": "Power Platform Export Data",
+  "description": "Power Platform export data from environment with provided schema",
+  "author": "Microsoft",
+  "helpMarkDown": "[More Info](https://aka.ms/buildtoolsdoc)",
+  "category": "Deploy",
+  "visibility": [
+    "Build",
+    "Release"
+  ],
+  "demands": [
+    "PowerPlatformTool"
+  ],
+  "version": {
+    "Major": 0,
+    "Minor": 1,
+    "Patch": 0
+  },
+  "preview": false,
+  "minimumAgentVersion": "1.95.0",
+  "instanceNameFormat": "Power Platform Export Data $(message)",
+  "inputs": [
+    {
+      "name": "authenticationType",
+      "type": "radio",
+      "label": "Authentication type",
+      "defaultValue": "PowerPlatformEnvironment",
+      "required": true,
+      "helpMarkDown": "Authentication via username/password is simpler to set up but won't work if your AAD tenant requires MFA; use Service Principal Name (SPN) if MFA is required.",
+      "options": {
+        "PowerPlatformEnvironment": "Username/password (no MFA support)",
+        "PowerPlatformSPN": "Service Principal/client secret (supports MFA)"
+      }
+    },
+    {
+      "name": "PowerPlatformEnvironment",
+      "type": "connectedService:Generic",
+      "label": "Service connection",
+      "required": true,
+      "visibleRule": "authenticationType = PowerPlatformEnvironment",
+      "helpMarkDown": "Authenticate with your Power Platform environment with username/password. Does not support MFA."
+    },
+    {
+      "name": "PowerPlatformSPN",
+      "type": "connectedService:powerplatform-spn",
+      "label": "Service connection",
+      "required": true,
+      "visibleRule": "authenticationType = PowerPlatformSPN",
+      "helpMarkDown": "Authenticate with your Power Platform environment with an Azure AppID, tenantID and client secret."
+    },
+    {
+      "name": "SchemaFile",
+      "label": "Schema File",
+      "type": "string",
+      "required": true,
+      "helpMarkDown": "Schema file name. It can be created using Configuration Migration Tool."
+    },
+    {
+      "name": "DataFile",
+      "label": "Data File",
+      "type": "string",
+      "defaultValue": "data.zip",
+      "required": true,
+      "helpMarkDown": "File name for data zip file. Default data.zip."
+    },
+    {
+      "name": "Overwrite",
+      "label": "Overwrite",
+      "type": "boolean",
+      "required": false,
+      "helpMarkDown": "Allow overwrite output data file if it already exists."
+    },
+    {
+      "name": "Verbose",
+      "label": "Verbose",
+      "type": "boolean",
+      "required": false,
+      "helpMarkDown": "Output more diagnostic information during data import/export."
+    },
+    {
+      "name": "Environment",
+      "label": "Environment",
+      "type": "string",
+      "required": false,
+      "helpMarkDown": "Environment (id, org id, url, unique name or partial name)."
```

Comment:
for consistency with all other tasks (and to avoid CX confusion), stick with the same scheme we use in other tasks: only mention 'Environment Url" but no unique/friendly name nor envId).
Not all tasks (even on the CLI side) do fully support the envid/envurl pair, and pretty much no verb yet has proper support for the full 4-tuple of envid/url/friendly/domain.

e.g. https://github.com/microsoft/powerplatform-build-tools/blob/0d378a5cefb6de022ccaee1d6851d54930dab5b6/src/tasks/import-solution/import-solution-v2/task.json#L60

```
    {
      "name": "Environment",
      "label": "Environment Url",
      "type": "string",
      "defaultValue": "$(BuildTools.EnvironmentUrl)",
      "required": false,
      "helpMarkDown": "Environment url this task targets; default is the pipeline variable that e.g. the CreateEnvironment task has set before."
    },
```

More important, all instance tasks must have the default value using the above pipeline var